### ISSUE TITLE
Add error pages for various response codes

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.org#573]
- [] 404 Not Found
- [] 418 "I'm a teapot."

**Comment by jeremyboggs** [comment]
@amandavisconti @lam2c I could use some help determining what to put on a 404 page. Could we work on this together?

**Comment by amandavisconti** [comment]
Yes! I've got some time on our calendars 1/10 and 1/15 to work on site stuff, but can also meet sooner.

Are there particular things yall have seen elsewhere and liked? 

Ideas: search field, links to frequently desired pages, link to email us for a consult or to report a missing page

**Comment by amandavisconti** [comment]
Added stuff from our conversations to the draft PR:
- mailto: email link + copyable plaintext SLab email address  
- sitemap link  
- newsletter signup embed  
- link to add issue to repo  
- search page link  
- FAQ page link

**Comment by amandavisconti** [comment]
I've merged the basic 404 page. Leaving this open, as I think @jeremyboggs has some ideas for further work and a 418?
